Document: Show default values of variables, with color previews

plastic041 opened the pull request:
Default values are wrapped in `<dd class="italic">`.
Shows color values with preview box(`<span class="color-preview">`).
Color previews are inline-styled using `--var()`.

Preview:
<img width="639" alt="terminal.css document of css variables, with default values written" src="https://user-images.githubusercontent.com/17811025/150621937-c7a9cca4-4e4e-4c9d-8182-14b87626bd04.png">

Gioni06 commented:
This is a really nice way to document the colors!! I just refactored the build process and moved away from ejs templates. I could go ahead and include your contribution manually, or you could go ahead and include the changes into the new template?

plastic041 commented:
I changed the file from ejs to `documentation.html` and added `--block-background-color` which was not there when I made the PR.

Also wrapped the values with `<code>` to differentiate them from the "Default: " part, but you can remove it if you don't want it.

<img width="674" alt="Before" src="https://github.com/Gioni06/terminal.css/assets/17811025/85858240-4565-42cc-a228-17f1cd412d34">

Before

<img width="674" alt="After" src="https://github.com/Gioni06/terminal.css/assets/17811025/8bbeec3a-9d37-42ac-9229-3bbcea12eb2e">

After

Gioni06 commented:
That looks unreasonably cool and is actually useful :) Thanks!

plastic041 commented:
I forgot when I made the PR, but can my name(@plastic041) be added to the "contributors" of the page?

Gioni06 commented:
Sure thing!

Am So., 26. Nov. 2023 um 01:46 Uhr schrieb Byeon Jaewoo <
***@***.***>:

> I forgot when I made the PR, but can my ***@***.***
> <https://github.com/plastic041>) be added to the "contributors" of the
> page?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Gioni06/terminal.css/pull/34#issuecomment-1826455085>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAEDCEOVE562WQGJC7AVK2TYGKGNRAVCNFSM5MRDJO2KU5DIOJSWCZC7NNSXTN2JONZXKZKDN5WW2ZLOOQ5TCOBSGY2DKNJQHA2Q>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>
